index.html

```diff
@@ -32,16 +32,16 @@ <h1>Le Catering</h1>
       <div class="aboutTextBox">
         <h2 class="centeredh2">About Catering</h2>
```

Comment:
Correct the class name to not include tag name in it.

Reply:
class name changed to 'centeredHead'.